test/functional/configuration/configuration.spec.js

```diff
@@ -393,6 +424,14 @@ test("shows error for duplicate IMS org ID", async t => {
   await instances[1].orgIdField.expectError(t);
 });
 
+test("shows error for invalid download link qualifier", async t => {
+  await extensionViewController.init(t, defaultInitInfo);
+  await instances[0].downloadLinkQualifierField.clear(t);
+  await instances[0].downloadLinkQualifierField.typeText(t, "[");
+  await extensionViewController.expectIsNotValid(t);
+  await instances[0].downloadLinkQualifierField.expectError(t);
+});
+
```

Comment:
I still think it would be good to test that downloadLinkQualifierField doesn't get flagged as invalid if clickCollectionEnabledField is unchecked. Do you agree?